### PR TITLE
added faction reputation info

### PR DESCRIPTION
**Pull request by ccMatrix**

### CODE REVIEW

**Review by Iliasgx — APPROVED** [review]
Looking good

File: faction_reputation/federation.json [hunk 1]
```diff
@@ -0,0 +1,177 @@
+[
+  {
+    "dailies_barrier": false,
+    "locked": false,
+    "points_min": -2000000,
+    "reputation_id": 1,
+    "reputation_name": ""
```
Comment (WebSpider) [inline faction_reputation/federation.json:7]: There is a field empty here, same for rom and klings

**Review by WebSpider — CHANGES_REQUESTED** [review]
please add the missing name for max negative rom/fed/kling rep :)

**Comment by ccMatrix** [comment]
I don't see any name in-game for that status. But what I found was that my game says -2000000 to hostile but the lcars data says enemy is the next level... We could pick any name you want for the lowest level

File: faction_reputation/federation.json (hunk 1 above)
Comment (ccMatrix) [reply faction_reputation/federation.json:7]: There is no name in the game for that level. the old bot called it `max negative` but that felt kinda strange so I just omitted the name for that level. If you have an idea for a good name I can add that instead.

**Comment by ccMatrix** [comment]
Screenshot_20210723-133511_Star Trek Fleet Command
And I think the old data has an error with the lowest level. My game tells me that the first named level after -2000000 is HOSTILE but the data says ENEMY is the next level. Maybe they are in the wrong order which we would need to fix manually.

**Comment by WebSpider** [comment]
The icon for roms on the right says "ENEMY" as well ... hmm.

File: faction_reputation/federation.json (hunk 1 above)
Comment (WebSpider) [reply faction_reputation/federation.json:7]: I've created an issue for it, lets solve this later.

**Review by WebSpider — APPROVED** [review]

**Comment by ccMatrix** [comment]
> The icon for roms on the left says "ENEMY" as well ... hmm.

Good catch. Might as well call it ENEMY then

**Pull request merged**